Sources/App/Models/Seeds/Development/ProfileProfileTypeSeed.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Profile_ProfileTypeSeed.swift
+//  App
+//
+//  Created by  LaptopGCampos on 7/11/18.
+//
+
+import FluentPostgreSQL
+
+struct ProfileTypePivotSeed: Migration {
+
+    typealias Database = PostgreSQLDatabase
```

Comment:
yes is needed, since Migration protocol needs that you provide what kind of database you will use so if I want to change for example, to  MySQL  I will  just need change this to the MySQL abstraction.

Reply:
Migration protocols use this internal, needs that you provide what type of database are you using for create the correct SQL code for the migration. this needed since is a requirement of the protocol.

Reply:
the connection doesn't provide the type.